### ISSUE TITLE
Enable keyboard control for sound editor.

### ISSUE THREAD

**Issue by quachtina96** [LLK/scratch-gui#908]
### Expected Behavior
Users should be able to tab along all buttons and inputs in the sound editor. By pressing 'space' or 'Enter' the can activate a button.

### Actual Behavior
None of the effects buttons can be tabbed to and activated using solely the keyboard.

### Operating System and Browser

Mac OS 10.12.6 Chrome 62.0.3202.89
